src/interaction.service.ts

```diff
@@ -0,0 +1,29 @@
+import { AxiosInstance } from "axios";
+import { Interactions, InteractionParams, RespondInteraction } from "./types/interactions";
+
+const endpoint = 'locations';
+//"TG9jYXRpb246MTI2NzE5" id del ejemplo del api
+//"TWVkaWFGaWxlOjEzMDMzMg==" id de locations
+
+export default (axios: AxiosInstance) => {
+  return {
+    getInteractions:(locationId:string, params: InteractionParams) => {
+      return axios.get(`${endpoint}/${locationId}/reviews`, {params});
+    },
+    getInteractionById:(params:Interactions)=>{
```

Comment:
this should get a parameter of type Array<string> and the param interactionIds should be set inside the function, in this way the developer that implement this functionality only need to send something like Interactions.getInteractionsById(['id1', 'id2']) instead of calling Interactions.getInteractionsById({interactionIds:['id1', 'id2']})